x-pack/legacy/plugins/ml/public/components/navigation_menu/navigation_menu.tsx

```diff
@@ -5,42 +5,55 @@
  */
 
 import React, { Fragment, FC } from 'react';
-import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule } from '@elastic/eui';
 
 // @ts-ignore
 import { isFullLicense } from '../../license/check_license';
 
 import { TopNav } from './top_nav';
+import { MainTabs } from './main_tabs';
 import { Tabs } from './tabs';
 
-const tabSupport = [
-  'jobs',
-  'settings',
-  'data_frames',
-  'data_frame_analytics',
-  'datavisualizer',
-  'filedatavisualizer',
-  'timeseriesexplorer',
-  'access-denied',
-  'explorer',
-];
+export type TabId = string;
+
+const tabSupport = {
```

Comment:
giving this a type of:
`type TabSupport = Record<string, string | null>;`
means you don't need type assertion on lines 41 and 43.